cdc/owner/gc_manager.go

```diff
@@ -49,7 +56,7 @@ type gcManager struct {
 	lastUpdatedPdTime   time.Time
 }
 
-func newGCManager() *gcManager {
+func newGCManager() GcManager {
```

Comment:
what is the benefit return interface instead of struct ?

Reply:
I got it, we should `take in terface and return struct`